# Toolbox - PoseRPY
# -------------------------------
# Description:
#   Message-type for Pose RPY (roll, pitch, yaw).
#   A more user friendly representation of pose, where orientation is represented as euler-angles
#
# Version:
#   0.1 -   Initial version
#           [18.12.2023] - Jan T. Olsen
# -------------------------------

# Pose RPY
geometry_msgs/Point position      # Position [x, y, z]
geometry_msgs/Point orientation   # Orientation [rx, ry, rz]
